### PR TITLE
Vood 70 png export

### PR DESCRIPTION
**Pull request by PascalSchuermannOst**

### CODE REVIEW

File: src/webview/panel-views/panelViewProxy.ts [hunk 1]
```diff
@@ -3,9 +3,15 @@ import { DebugSessionProxy } from '../../debug-adapter/debugSessionProxy';
...
 
 export interface UpdatePanelViewCommand {
   [key: string]: unknown;
   command: string;
 }
+
+export interface ExportPanelViewCommand {
+  [key: string]: unknown;
+  command: string;
+}
```
Comment (GinoCardillo-OST) [inline src/webview/panel-views/panelViewProxy.ts:17]: Combine those two and create an interface `PanelViewCommand`.

File: package.json [hunk 2]
```diff
@@ -22,6 +23,11 @@
         "command": "visual-oo-debugger.openDebugger",
         "title": "Open debugger view",
         "category": "VOOD"
+      },
+      {
+        "command": "visual-oo-debugger.exportPNG",
+        "title": "export PNG",
```
Comment (GinoCardillo-OST) [inline package.json:29]: "Export as PNG"

File: package.json [hunk 3]
```diff
@@ -22,6 +23,11 @@
         "command": "visual-oo-debugger.openDebugger",
         "title": "Open debugger view",
         "category": "VOOD"
+      },
+      {
+        "command": "visual-oo-debugger.exportPNG",
```
Comment (GinoCardillo-OST) [inline package.json:28]: Maybe you could show this command only if the panel is opened.

**Review by GinoCardillo-OST — APPROVED** [review]

File: src/webview/html/visjsDebuggerPanel.html [hunk 4]
```diff
@@ -21,11 +21,18 @@
...
           const container = document.querySelector('#visjs-network');
           const network = new vis.Network(container, message.data, message.options);
         }
+        if (message.command === 'exportVisjs') {
+          const canvas = document.querySelector('#visjs-network canvas');
+          const img = canvas.toDataURL('image/png');
+          var downloadLink = document.createElement('a');
+          downloadLink.download = 'export.png';
+          downloadLink.href = img;
+          downloadLink.click();
```
Comment (alexandre-lagadec-ost) [inline src/webview/html/visjsDebuggerPanel.html:34]: This is the first time I have seen this pattern without [the `a` element being added to and removed from the DOM](https://stackoverflow.com/questions/19327749/javascript-blob-filename-without-link).
However, it seems to be working.

**Review by alexandre-lagadec-ost — APPROVED** [review]

**Review by alexandre-lagadec-ost — APPROVED** [review]

**Pull request merged**